server/routerlicious/packages/routerlicious-base/src/alfred/runnerFactory.ts

```diff
@@ -264,89 +260,163 @@ export class AlfredResourcesFactory implements core.IResourcesFactory<AlfredReso
 			);
 		};
 
+		const tenantThrottlersMap = new Map<string, string>();
+		const tenantGroup1 = config.get("alfred:throttlingGroup:tenantGroup1") as string[];
+		tenantGroup1.forEach((t) => tenantThrottlersMap.set(t, "tenantGroup1"));
+
```

Comment:
I agree with Zach here. We will need to make this design more flexible as right now tenantGroup1 is hardcoded which means we can only have 1 group and also, we cannot rename the group to make it more readable. @tianzhu007 could you please add a backlog workitem for this? Thank you :)

Reply:
Sounds good! Made a work item here
https://msazure.visualstudio.com/One/_workitems/edit/21283047